src/visualization/convert.jl

```diff
@@ -73,6 +73,54 @@ function get_data_2d(center_level_0, length_level_0, leaf_cells, coordinates, le
 end
 
 
+# Extract data from a 1D DG solution and prepare it for visualization as a line plot.
+# This returns a tuple with
+# - x coordinates
+# - nodal 1D data
+#
+# Note: This is a low-level function that is not considered as part of Trixi's interface and may
+#       thus be changed in future releases.
+function get_data_1d(original_nodes, unstructured_data, nvisnodes)
+  # Get the dimensions of u; where n_vars is the number of variables, n_nodes the number of nodal values per element and n_elements the total number of elements.
+  n_nodes, n_elements, n_vars = size(unstructured_data)
+
+  # Set the amount of nodes visualized according to nvisnodes.
+  if nvisnodes === nothing
+    max_nvisnodes = 2 * n_nodes
+  elseif nvisnodes == 0
+    max_nvisnodes = n_nodes
+  else
+    max_nvisnodes = nvisnodes
+  end
+
+  interpolated_nodes = Array{Float64,2}(undef, max_nvisnodes, n_elements)
+  interpolated_data = Array{Float64,3}(undef, n_vars, max_nvisnodes, n_elements)
```

Comment:
```suggestion
  interpolated_data = Array{Float64,3}(undef, max_nvisnodes, n_elements, n_vars)
```
Together with the changes below, this makes the code a little more readable (I think) and avoids the transpose